inginious/frontend/parsable_text.py

```diff
@@ -35,6 +35,8 @@ def run(self):
         if not self.content:
             translation = _get_inginious_translation()
             self.content = [translation.gettext("[no content]")]
+        
+        self.options.setdefault('classes', []).append('theme={}'.format(flask.current_app.user_manager.get_codemirror_theme()))
```

Comment:
That should be OK but maybe it's more careful and idiomatic (it seems to be the de-facto rule) to use `-` instead of `=` to separate the prefix of your CSS class. 

In case of bad HTML, browsers typically try to fix the error by themselves. A `=` is part of the HTML tag syntax and may be badly interpreted as an attribute in case a correction is needed. Of course we aim at providing valid HTML to the client, but those things can still happen.